### PR TITLE
Add basic rpm-ostree compose test 

### PR DESCRIPTION
**Pull request by mike-nguyen**
Since the ability to create composes exists for Fedora and Centos
Atomic Host Continuous streams, we should have a test for it.

This change only includes a basic test for rpm-ostree compose.
There should be more tests to cover additional rpm-ostree compose
subcommands and options.

### CODE REVIEW

File: tests/rpm-ostree/main.yml [hunk 1]
```diff
@@ -836,3 +836,170 @@
...
+            repo_name: 'centos'
+            base_json: 'centos-atomic-host.json'
+            json_file: 'centos-atomic-host-continuous.json'
+
+        # CentOS Atomic Host runs out of space in the root partition when
+        # creating a tree and rebasing to it so the root partition needs
+        # to be increased
+        - name: Resize partition for CentOS
+          when: ansible_distribution == "CentOSDev"
+          command: lvextend -r -L 6G atomicos/root
```
[the remark this answers is not in the document]
Comment (mike-nguyen) [reply tests/rpm-ostree/main.yml]: I thought I pushed this in my fixup but it's still in my local checkout.   It'll be there in my next fixup!

File: tests/rpm-ostree/main.yml [hunk 2]
```diff
@@ -836,3 +836,170 @@
...
+          command: >
+            docker run -i --privileged -v {{ base_dir }}:{{ base_dir }}:z
+              docker.io/miabbott/aht-tools /bin/bash -c
+              "git clone {{ repo_url }} {{ base_dir }}/{{ repo_name }};
+               cd {{ base_dir }}/{{ repo_name }}; git checkout f27"
+
+        - name: Clone sig-atomic-buildscripts repo
+          when: ansible_distribution == "CentOSDev"
+          command: >
+            docker run -i --privileged -v {{ base_dir }}:{{ base_dir }}:z
```
[the remark this answers is not in the document]
Comment (mike-nguyen) [reply tests/rpm-ostree/main.yml]: I just tried it without and those flags are unnecessary.  I think I was playing around with the container and copy and pasta the command over

**Comment by mike-nguyen** [comment]
Just pushed a fixup with fixes to address the comments.  I also added a rollback, reboot, and cleanup to bring the system back to the original deployment.

**Comment by miabbott** [comment]
Ran this against RHELAH, CAHC, F27 and all good.  Great job!

**Pull request merged**